etna/pipeline/base.py

```diff
@@ -24,6 +25,89 @@
 from etna.metrics import Metric
 from etna.metrics import MetricAggregationMode
 
+Timestamp = Union[str, pd.Timestamp]
+
+
+class CrossValidationMode(Enum):
+    """Enum for different cross-validation modes."""
+
+    expand = "expand"
+    constant = "constant"
+
+
+class FoldMask(BaseMixin):
+    """Container to hold the description of the fold mask.
+    Fold masks are expected to be used for backtest strategy customization.
```

Comment:
May be we should add empty line here according to https://numpydoc.readthedocs.io/en/latest/format.html
Example that I've found: https://github.com/numpy/numpy/blob/v1.22.0/numpy/lib/nanfunctions.py#L953-L1056